repose-aggregator/services/phone-home/src/main/scala/org/openrepose/core/services/phonehome/PhoneHomeService.scala

```diff
@@ -127,7 +128,23 @@ class PhoneHomeService @Inject()(@Value(ReposeSpringProperties.CORE.REPOSE_VERSI
         }
       }
 
+      def getCurrentFormattedDateTime(): String = {
+        // From http://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html
+        // "For formatting, if the offset value from GMT is 0, 'Z' is produced."
+        // This only manipulates the Time Zones that produce a 'Z'.
+        val formattedString = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX").format(new Date())
+        if (formattedString.endsWith("Z")) {
+          formattedString.substring(0, formattedString.length() - 1) + "+00:00"
+        } else {
+          formattedString
+        }
```

Comment:
If this service is primarily for internal consumption, should we just set the time zone to our time zone instead? That way we don't have to deal with user time zones, and we don't have to worry about converting the Z.